Makefile

```diff
@@ -108,29 +99,21 @@ $(haskell_kompiled): $(haskell_defn)
 	    --main-module $(MAIN_MODULE) --syntax-module $(MAIN_SYNTAX_MODULE) $< \
 	    $(KOMPILE_OPTIONS)
 
-$(java_kompiled): $(java_defn)
-	$(K_BIN)/kompile --backend java                                           \
-	    --directory $(java_dir) -I $(java_dir)                                \
-	    --main-module $(MAIN_MODULE) --syntax-module $(MAIN_SYNTAX_MODULE) $< \
-	    $(KOMPILE_OPTIONS)
-
 # Testing
 # -------
 
 TEST  := ./kwasm
 CHECK := git --no-pager diff --no-index --ignore-all-space -R
 
-TEST_CONCRETE_BACKEND       := llvm
-TEST_FLOAT_CONCRETE_BACKEND := java
-TEST_SYMBOLIC_BACKEND       := haskell
-
-tests/proofs/memory-concrete-type-spec.k%prove: KPROVE_MODULE=MEMORY-CONCRETE-TYPE-LEMMAS
-tests/proofs/wrc20-spec.k%prove: KPROVE_MODULE=WRC20-LEMMAS
-tests/proofs/locals-spec.k.prove: TEST_SYMBOLIC_BACKEND=haskell
+TEST_CONCRETE_BACKEND := llvm
+TEST_SYMBOLIC_BACKEND := haskell
 
 KPROVE_MODULE := KWASM-LEMMAS
 
-PROVE_OPTIONS ?=
+KPROVE_OPTIONS ?=
+
+tests/proofs/memory-concrete-type-spec.k%prove: KPROVE_MODULE = MEMORY-CONCRETE-TYPE-LEMMAS
+tests/proofs/wrc20-spec.k%prove:                KPROVE_MODULE = WRC20-LEMMAS
```

Comment:
You can remove the `%` here I think, since there is no longer a `klab-prove` target.

Reply:
Done